niworkflows/viz/utils.py

```diff
@@ -283,28 +283,32 @@ def plot_segs(image_nii, seg_niis, out_file, bbox_nii=None, masked=False,
 
 def _plot_anat_with_contours(image, segs=None, compress='auto',
                              **plot_params):
-    assert segs is not None
-
+    nsegs = len(segs or [])
     plot_params = plot_params or {}
-    colors = plot_params.pop('colors', []) or []  # colors should not be None
-    nsegs = len(segs)
+    colors = plot_params.pop('colors', None) or []
+    levels = plot_params.pop('levels', None) or []
```

Comment:
Suspect I'm missing why you changed it, but isn't this equivalent to:

```Python
colors = plot_params.pop('colors', [])
levels = plot_params.pop('levels', [])
```

Reply:
Unfortunately not:
```Python
>>> plot_params = {'colors': None, 'levels': None}
>>> colors = plot_params.pop('colors', [])
... levels = plot_params.pop('levels', [])
>>> colors is None
True
>>> levels is None
True
```

Many times they are set and `None`, so I had to change to that ugly `or`. Maybe there's a better way of doing this I'm missing (e.g. cleaning the dictionary of `None`s before, however I'm not sure other kw may use None as a valid value or maybe replacing and then pop... I don't have a preference).

Reply:
Got it. Maybe just a quick comment (e.g. "None is a valid parameter") so the next person to look at this won't think the same thing.